src/CrudTrait.php

```diff
@@ -191,4 +191,215 @@ public function uploadMultipleFilesToDisk($value, $attribute_name, $disk, $desti
 
         $this->attributes[$attribute_name] = json_encode($attribute_value);
     }
+
+    /**
```

Comment:
On a general note this function strikes me as being too long; personally I'd split it into a few protected (or public?) methods with easy to understand names...